public/doc/openapi-3-v1.0.0.json

```diff
@@ -14,7 +14,9 @@
   },
   "security": [
     {
-      "BasicAuth": []
+      "BasicAuth": [
```

Comment:
Is this the recommended way over an inline `[]` declaration?

Reply:
@syncrou Yes, the arrays are always split into separate lines

Reply:
@syncrou This is JSON.pretty_generate output, its not hand written, we had several alignment issues.